pom.xml

```diff
@@ -562,10 +625,10 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-compiler-plugin</artifactId>
                         <configuration>
-                            <source>1.8</source>
-                            <target>1.8</target>
-                            <testSource>1.8</testSource>
-                            <testTarget>1.8</testTarget>
+                            <source>11</source>
```

Comment:
This configuration already exists in lines 242ff

Reply:
You are right. I will remove it. Do we still need the profile itself?